### ISSUE TITLE
Exported PCAP file corrupted.

### ISSUE THREAD

**Issue by pbooysen** [sipcapture/homer#298]
We are running Homer using the Docker images provided on Ubuntu 16.04. When exporting calls the resulting pcap file cannot be opened in Wireshark.

Investigating the issue it is found that a line feed character (0A) has been inserted at the start of the file. This is also the case with TXT exports. 

We have Homer running on an older Ubuntu 14.04 system also using docker where this issue does not occur. I suspect this is a version issue somewhere in software.

Is there anything I can check on my side to determine where the issue is originating from?

OS:
  Ubuntu 16.04.4
Docker:
  Server Version: 18.03.1-ce
  Storage Driver: overlay2
  Kernel Version: 4.4.0-127-generic

**Comment by lmangani** [comment]
Hi @pbooysen could you check the GIT version in use on the working vs. non-working versions locally?
```
git rev-parse HEAD
```

**Comment by pbooysen** [comment]
Hi @lmangani this is the output for the homer-docker repo.

The working version is d447c86c65176c793d29aa38d7768d08d54cb383
and the faulty one is 5b62b401a12002f8bcbf0c2ed5159f927f4af4f3

**Comment by pbooysen** [comment]
I did some more digging to see if I can spot where the linefeed is being inserted into the pcap export.

From what I can see the pcap is being generated by RestApi/Search.php so I decided to replace that file with the working copy to see if the outcome changes, however the linefeed is still present in the export after the change.

After that I counted all the linefeed (\n) occurrences in the 2 files and they are similar. With this in mind I am not sure if this is an API issue.
